sys/vm/vm_fault.c

```diff
@@ -170,6 +170,12 @@ SYSCTL_INT(_vm, OID_AUTO, pfault_oom_wait, CTLFLAG_RWTUN,
     "Number of seconds to wait for free pages before retrying "
     "the page fault handler");
 
+static bool capstore_also_dirty = 0;
+SYSCTL_BOOL(_vm, OID_AUTO, capstore_also_dirty, CTLFLAG_RW,
+    &capstore_also_dirty, 0,
+    "When marking a page CAPSTORE, also mark it CAPDIRTY, "
```

Comment:
I'd prefer to see this in the pmap so it's more obviously the old behaviour

Reply:
Just to clarify: you mean you want a `sysctl` that causes `pmap_enter` to interpret `PGA_CAPSTORE` as if `PGA_CAPDIRTY` were also set?

Reply:
After reminding myself of the pmap diff, yes, I believe so (i.e. `PTE_CW` reverts to implying `PTE_CD` as it currently does in CheriBSD).

Reply:
(Which amounts to the same thing, but it's a "oncer per caller of `pmap_enter`" vs "once per pmap implementation" thing, and the former could very easily end up missing something, especially in a merge, whereas the latter is trivially correct and not likely to break)